.github/workflows/pull-request-queue.yaml

```diff
@@ -14,9 +14,19 @@ jobs:
     name: enqueue
     runs-on: ubuntu-20.04
     steps:
+      - name: Query vault
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          exportEnv: false
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
+          secrets: |
+            secret/data/kubecf "github.cfcontainerizationbot-labeler-personal-access-token" | GH_BOT_TOKEN
```

Comment:
I believe we can use `github-access-token` instead (i.e. @suse-cf-ci-bot) instead of this one (which would be @cfcontainerizationbot).

There's also`github-limited-token` but I'm not sure what the differences in permission are; might be useful to try and see if that is enought to set labels.

Reply:
Addressed with commit abcd4410dac0c7a0970ad5ea69f6b01bbf05a048